app/views/idv/doc_auth/link_sent.html.erb

```diff
@@ -47,4 +47,4 @@
   <% end %>
 <% end %>
 
-<%= render 'idv/doc_auth/back', action: 'cancel_link_sent' %>
+<%= render 'idv/doc_auth/back', action: 'cancel_link_sent', class: 'doc_capture_back_link' %>
```

Comment:
I feel like we have been using more kebab case class names, should we kebab-case this and `doc_capture_continue_button_form` while we're here? (I don't feel strongly)

Reply:
> I feel like we have been using more kebab case class names, should we kebab-case this and `doc_capture_continue_button_form` while we're here? (I don't feel strongly)

That's a good call. I'd just instinctively followed consistent with the other class, but in general I'd agree dashes would be better here than underscores.

Reply:
> I feel like we have been using more kebab case class names, should we kebab-case this and `doc_capture_continue_button_form` while we're here? (I don't feel strongly)

Renamed in c7830629a.